### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by JantuDeb**
Added responsiveness to sidebar

### CODE REVIEW

File: documentation/styles.css [hunk 1]
```diff
@@ -108,3 +108,36 @@ header {
...
 }
+
+@media screen and (max-width: 40em) {
+  aside.sidebar {
+    overflow-y: hidden;
+    overflow-x: auto;
+    flex-basis: auto;
+    height: fit-content;
+    width: 100vw;
+    padding: 0.5rem 0.5rem;
```
Comment (ajit-sharmaGH) [inline documentation/styles.css:119]: I think it can be reduced to 0.5rem only

**Review by ajit-sharmaGH — APPROVED** [review]
Everything looks fine to me

File: documentation/styles.css (hunk 1 above)
Comment (JantuDeb) [reply documentation/styles.css:119]: Yes, thanks for pointing this

**Review by psychoactiv — APPROVED** [review]
Good to merge

**Pull request merged**